index.js

```diff
@@ -17,349 +21,408 @@ var cluster = new couchbase.Cluster(
     username: 'Administrator',
     password: 'password'
   }
-);
+)
 
 // Open a specific Couchbase bucket, `travel-sample` in this case.
-var bucket = cluster.bucket('travel-sample');
-// And select the default collection
-var coll = bucket.defaultCollection();
+var bucket = cluster.bucket('travel-sample')
 
 // Set up our express application
-var app = express();
-app.use(cors());
-app.use(bodyParser.json());
+var app = express()
+app.use(morgan('dev'))
+app.use(cors())
+app.use(express.json())
+
+app.use('/api-docs', swaggerUi.serve, swaggerUi.setup(swaggerDocument));
```

Comment:
Should we have this as `/apidocs` or `/api-docs`? Just thinking it would be good to keep it consistent across projects. Currently I've got it as `/apidocs` in Python because the library I used defaults to that, but I can try to configure it if needed.

Reply:
sorry, I just copy/pasted the Node library example too! I'll change to match